neurodsp/plts/spectral.py

```diff
@@ -18,9 +18,9 @@ def plot_power_spectra(freqs, powers, labels=None, colors=None, ax=None, **kwarg
 
     Parameters
     ----------
-    freqs : 1d array or list of 1d array
+    freqs : 1d or 2d array
         Frequency vector.
-    powers : 1d array or list of 1d array
+    powers : 1d or 2d array
```

Comment:
Since a list of 1d array inputs still works right, I think we want to keep this listed in the docs, so I think it should probably say something like:
`powers : 1d or 2d array or list of 1d array`

Same for freqs above, I think.